ui/lib/core/addon/utils/client-count-utils.js

```diff
@@ -193,14 +184,11 @@ export const namespaceArrayToObject = (totalClientsByNamespace, newClientsByName
       };
     });
 
-    const { label, clients, entity_clients, non_entity_clients, secret_syncs, new_clients } = namespaceObject;
```

Comment:
Did my best to clean up instances of destructuring so adding future client types is less cumbersome